templates/editor/editor-common.jade

```diff
@@ -0,0 +1,100 @@
+extends ../layout
```

Comment:
Please revert the merging of the editor template file. This is currently being dealt with in its own GCI task, since there is a lot less similarity here than in the other merged files.